mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown

```diff
@@ -145,3 +145,15 @@ resource blocks that contain both fields in a conflicting pair, and remove one o
 The fields that are removed from the configuration will still have Computed values,
 that are derived from the API.
 
+## Resource: `google_compute_backend_service`
+
+### `enabled` is the new and only required field for message IAP
```

Comment:
```suggestion
### `iap.enabled` is now required in the `iap` block
```

Reworded to be more Terraform-specific? Let me know what you think

Reply:
I think we can make it optional. In that case, the rest of the fields under IAP will be ignored but the request would still succeed.